systemd-swap

```diff
@@ -1,613 +1,730 @@
-        UNIT_NAME=${UNIT_PATH##*/}
-        swapoff "${DEV}"
-        rm -vf "${UNIT_PATH}"
-      fi
-    done
-
-    [[ -f ${WORK_DIR}/swapfc/.lock ]] && rm -f "${WORK_DIR}/swapfc/.lock"
-
-    echo "$swap_units" | while read -r UNIT_PATH; do
-      if grep -q swapfc "${UNIT_PATH}"; then
-        DEV=$(GET_WHAT_FROM_SWAP_UNIT "${UNIT_PATH}")
-        UNIT_NAME=${UNIT_PATH##*/}
-        swapoff "${DEV}"
-        rm -vf "${UNIT_PATH}"
-        [[ -f ${DEV} ]] && rm -f "${DEV}"
-      fi
-    done
-
-    echo "$swap_units" | while read -r UNIT_PATH; do
-      if grep -q zram "${UNIT_PATH}"; then
-        DEV=$(GET_WHAT_FROM_SWAP_UNIT "${UNIT_PATH}")
-        UNIT_NAME=${UNIT_PATH##*/}
-        INFO "Zram: swapoff ${DEV}"
-        swapoff "${DEV}"
-        rm -vf "${UNIT_PATH}"
-        zramctl -r "${DEV}"
-      fi
-    done
-    [[ -v ZRAM_ALREADY_SET ]] || modprobe -r zram
-
-    if [[ -d ${WORK_DIR}/zswap ]]; then
-      INFO "Zswap: restore configuration: start"
-      for file in "${WORK_DIR}/zswap"/*; do
-        read_l "${file}" > "${ZSWAP_M_P}/${file##*/}"
-      done
-      INFO "Zswap: restore configuration: complete"
-    fi
-
-    wait
-
-    rm -f "${LOCK_STARTED}"
-  ;;
-  status)
-    declare -a swap_stats
-    get_mem_stat_multi swap_stats SwapTotal SwapFree
-    SWAP_USED=$(( swap_stats[0] - swap_stats[1] ))
-    unset swap_stats
-
-    if [[ -d /sys/module/zswap ]]; then
-      echo Zswap:
-      read -r USED_BYTES < /sys/kernel/debug/zswap/pool_total_size
-      USED_PAGES=$(( USED_BYTES / PAGE_SIZE ))
-      read -r STORED_PAGES < /sys/kernel/debug/zswap/stored_pages
-      STORED_BYTES=$(( STORED_PAGES * PAGE_SIZE ))
-      RATIO=0
-      (( STORED_PAGES > 0 )) && RATIO=$(( USED_PAGES * 100 / STORED_PAGES ))
-      {
-        grep . /sys/module/zswap/parameters/* | \
-          cut -d'/' -f6 | \
-          sed -e 's/:/ /g' -e 's/^/. /g'
-      } | column -t
-      {
-        grep . /sys/kernel/debug/zswap/* | \
-          cut -d'/' -f6 | \
-          sed -e 's/:/ /g' -e 's/^/. . /g' | column -t
-        echo . . compress_ratio ${RATIO}%
-        (( SWAP_USED > 0 )) && \
-          echo . . zswap_store/swap_store ${STORED_BYTES}/${SWAP_USED} $(( STORED_BYTES * 100 / SWAP_USED ))%
-      } | column -t
-    fi
-
-    if zramctl | grep -q '\[SWAP\]'; then
-      echo ZRam:
-      zramctl | \
-        grep -e '^NAME\|\[SWAP\]' | \
-        sed -e 's/\(MOUNTPOINT\|\[SWAP\]\)$//g' -e 's/^/.\t/g' | \
-        column -t | \
-        uniq
-    fi
-
-    if [[ -d ${WORK_DIR}/swapd ]]; then
-      echo swapD:
-      swapon --raw | \
-        grep -v 'zram\|file\|loop' | \
-        sed 's/^/.\t/g' | \
-        column -t
-    fi
-
-    if [[ -d ${WORK_DIR}/swapfc ]]; then
-      echo swapFC:
-      swapon --raw | \
-        grep -e 'NAME\|file\|loop' | \
-        sed 's/^/.\t/g' | \
-        column -t
-    fi
-  ;;
-  *)
-    help
-  ;;
-esac
-
-exit 0
-# vim: set ft=sh ts=2 sw=2 et:
+#!/usr/bin/env -S python3 -u
```

Comment:
Actually, when looking at other Linux projects they all seem to use `#!/usr/bin/python{,3}` instead of `env`, I think we should follow that too

Reply:
Looks like `#!/usr/bin/env -S python3 -u` is the preferred solution. `env` is always there and it will most likely find `python3`.

See:
- https://stackoverflow.com/a/19305076
- https://www.python.org/dev/peps/pep-0394/#for-python-script-publishers
- https://mail.python.org/pipermail/tutor/2007-June/054816.html

Reply:
The two python programs I have installed and running both use `/usr/bin/python{,3}` (tuned and firewalld).
The python.org pep linked only mentions using env for "scripts that are only expected to be run in an activated virtual environment" and since we're installing this as a system service this certainly isn't the case. 
![Htop screenshot](https://user-images.githubusercontent.com/26655508/87943046-c6128480-ca9d-11ea-93b6-2856a7a9a6e3.png)
Having `/usr/bin` also makes stuff like htop much more pleasing.
Since systemd-swap should be installed by the package manager and not pip it makes sense to use the system python installation which is always under `/usr/bin/python3`.

Reply:
Updated the shebang.